docs/docsite/versions.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  active:
+   - devel
+   - stable-2.18
+   - stable-2.17
+   - stable-2.16
+  eol:
+   - stable-2.15
```

Comment:
if this represents non-core dist, the versioning here would probably be different